pages/400.html

```diff
@@ -0,0 +1,33 @@
+
+<html>
+    <head>
+        <link type="text/css" rel="stylesheet" media="all" href="error_style/style.css">
+        <link type="text/css" rel="stylesheet" media="all" href="error_style/bootstrap_style.css">
+        <link type="text/css" rel="stylesheet" media="all" href="error_style/additional_style.css">
+    </head>
+    <body>
+        <section class="section section-primary section-lead">
+            <div class="container">
+                <div class="row">
+                    <div class="col-sm-6 flower-container">
+                        <div class="flower-home"></div>
+                    </div>
+                    <div class="col-sm-6 container-welcome">
+                        <div class="welcome-container-inner">
+                            <h1>Your client has submitted an invalid request</h1>
+                            <p>Http error code: 400</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <footer class="footer">
+            <div class="container">
+                <p class="text-copyright text-center">
+                    Copyright &copy; 2015 Dimagi, Inc.
+                    Read the <a href="#">End User License Agreement</a>.
```

Comment:
unless i'm missing something I don't think this will work without some javascript. could probably just remove it.